PowerVCF.psm1

```diff
@@ -2161,9 +2161,9 @@ Function Get-VCFLicenseKey
         This example shows how to get a specified License key
 
         .EXAMPLE
-        PS C:\> Get-VCFLicenseKey -productType "VCENTER,VSAN"
```

Comment:
it is not longer possible to specific multi product type ?

Reply:
You still pass multiple:  Get-VCFLicenseKey -productType "ESXI,NSXT"

Reply:
it will be better to set ProductType to [string[]]$productType in this case